jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
-`circleci orb validate /tmp/orb.yml`
+#### Step 4 - Validate your build configuration
+Once you have configured the CircleCI CLI, you then need to validate your build configuration to ensure you can correctly create or update an orb. You can find detailed instructions on how to validate your build configuration in the [Validating a Build Config] (https://circleci.com/docs/2.0/creating-orbs/#validating-a-build-config) section.
 
-5. Publish a development version of your orb, for example:
-`circleci orb publish /tmp/orb.yml sandbox/hello-world@dev:first`
+#### Step 5 - Set the Version Property to 2.1
+After validating your build configuration, set the version property to 2.1. More information on how to set the version property can be found in the [Set the Version Property to 2.1](https://circleci.com/docs/2.0/creating-orbs/#set-the-version-property-to-21) section on this page.
 
-6. Once you are ready to push your orb to production, you can publish it manually using ```circleci orb publish``` command or promote it directly from the development version. To increment the new dev version to become 0.0.1, use the following command:
-`circleci orb publish promote sandbox/hello-world@dev:first patch`
+#### Step 6 - Create a new Orb using inline template
+CircleCI provides you an example inline template that you can, and should, use when creating your own orbs. The example inline teamplat can be found in the [Example Inlline Template](https://circleci.com/docs/2.0/creating-orbs/#example-inline-template) section on this page.
 
-7. Your orb is now published in an immutable form as a production version and can be used safely in builds. You can view the source of your orb by using:
-`circleci orb source sandbox/hello-world@0.0.1`
+#### Step 7
+Now that you have copied the example inline template from step 6, you will need to add your own orb elements (Jobs, Commands, and Executors to this inline template. For more information on these orb elements, refer to the [Commands](https://circleci.com/docs/2.0/using-orbs/#commands), [Jobs](https://circleci.com/docs/2.0/using-orbs/#jobs), and [Executors](https://circleci.com/docs/2.0/using-orbs/#executors) sections found in the [Using Orbs](https://circleci.com/docs/2.0/using-orbs/#section=configuration) page.
 
-**Note:** Running `circleci setup`, providing a personal API token, and enabling [Build Processing]({{ site.baseurl }}/2.0/build-processing/) for your project are prerequisites for creating orbs.
+#### Steps 8 - 9 - Validating your Orb
+When you have finished authoring your orb, you need to validate your orb to ensure it will work on in your configuration before publication. CircleCI provides several different tools to validate your orb, including the `circleci/orb-tools` orb. For more information on using the `circleci/orb-tools` orb, see the [Validate and Publish Your Orb](https://circleci.com/docs/2.0/creating-orbs/#validate-and-publish-your-orb) section on this page.
```

Comment:
You can also just run the validate command from your CLI, which is much easier to do for a quick feedback loop. (You don't have to wait for a build, you don't have to navigate through a UI, you don't have to go dig up your orb from the registry or wherever you're viewing it.)

Orb-tools is really nice once you've gotten the gist of how to build an orb, it's a little more overhead for actually playing around with building your first orb.

Reply:
Re-worded this section to mention that using the validate command from the CLI is much easier